src/wp-includes/class-wp-customize-widgets.php

```diff
@@ -1310,12 +1310,10 @@ public function export_preview_data() {
 		foreach ( $settings['registeredWidgets'] as &$registered_widget ) {
 			unset( $registered_widget['callback'] ); // May not be JSON-serializeable.
 		}
-
-		?>
-		<script type="text/javascript">
-			var _wpWidgetCustomizerPreviewSettings = <?php echo wp_json_encode( $settings ); ?>;
-		</script>
-		<?php
+		wp_print_inline_script_tag(
+			// language=JavaScript
+			sprintf( 'var _wpWidgetCustomizerPreviewSettings = %s;', wp_json_encode( $settings ) )
```

Comment:
PhpStorm flags the sprintf placeholder as a syntax error:

> Expression expected

![image](https://github.com/WordPress/wordpress-develop/assets/134745/ac37a413-aeea-4db3-9b55-4ce6fee27bad)

Reply:
presumably because of the `// language=JavaScript` comment, which shouldn't be there since this is still PHP?